tests/NHSD.BuyingCatalogue.Ordering.Api.IntegrationTests/Features/Order Item Create.feature

```diff
@@ -0,0 +1,134 @@
+        | catalogue solution |
+        | additional service |
+
+@7840
+Scenario: 4. Create catalogue solution order item with invalid order ID
+    Given the user creates a request to add a new <ItemType> order item to the order with ID 'INVALID'
+    And the user enters the 'complete' create order item request payload
+    When the user sends the create order item request
+    Then a response with status code 404 is returned
+    Examples: Item Types
+        | ItemType           |
+        | catalogue solution |
+        | additional service |
+
+@7840
+Scenario: 5. If a user is not authorised then they cannot create a catalogue solution order item
+    Given no user is logged in
+    And the user creates a request to add a new <ItemType> order item to the order with ID 'C000014-01'
+    And the user enters the 'complete' create order item request payload
+    When the user sends the create order item request
+    Then a response with status code 401 is returned
+    Examples: Item Types
+        | ItemType           |
+        | catalogue solution |
+        | additional service |
+
+@7840
+Scenario: 6. A non buyer user cannot create a catalogue solution order item
+    Given the user is logged in with the Authority role for organisation 4af62b99-638c-4247-875e-965239cd0c48
+    And the user creates a request to add a new <ItemType> order item to the order with ID 'C000014-01'
+    And the user enters the 'complete' create order item request payload
+    When the user sends the create order item request
+    Then a response with status code 403 is returned
+    Examples: Item Types
+        | ItemType           |
+        | catalogue solution |
+        | additional service |
+
+@7840
+Scenario: 7. A buyer user cannot create a catalogue solution order item for an organisation they don't belong to
+    Given the user is logged in with the Buyer role for organisation e6ea864e-ef1b-41aa-a4d5-04fc6fce0933
+    And the user creates a request to add a new <ItemType> order item to the order with ID 'C000014-01'
+    And the user enters the 'complete' create order item request payload
+    When the user sends the create order item request
+    Then a response with status code 403 is returned
+    Examples: Item Types
+        | ItemType           |
+        | catalogue solution |
+        | additional service |
+
+@7840
+Scenario: 8. Service Failure
+    Given the call to the database will fail
+    And the user creates a request to add a new <ItemType> order item to the order with ID 'C000014-01'
+    And the user enters the 'complete' create order item request payload
+    When the user sends the create order item request
+    Then a response with status code 500 is returned
+    Examples: Item Types
+        | ItemType           |
+        | catalogue solution |
+        | additional service |
+
+@7840
+Scenario: 9. Create catalogue solution order item and the catalogue solution order section should be marked as complete
```

Comment:
This test will also need updating to include the additional service "completed" flag when it is introduced. Is it a good idea to create a task to track this?

Reply:
Possibly? Which story would that task go under though, a story that hasn't been brought in yet I assume. I'm inclined to leave it down to our memories, for during task breakdown tomorrow.

Reply:
👍  I am relying on your memory not mine 😄